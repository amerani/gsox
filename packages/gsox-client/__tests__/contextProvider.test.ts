import * as React from "react";
import { DataProvider } from "../";

test('should render', () => {
      expect(React.createElement(DataProvider)).toMatchSnapshot();
})

test('should render with props', () => {
      expect(React.createElement(DataProvider, { client:{ }, ws: { } })).toMatchSnapshot();
})