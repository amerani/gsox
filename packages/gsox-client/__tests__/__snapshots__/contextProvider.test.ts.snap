// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `<SubscriptionProvider />`;

exports[`should render with props 1`] = `
<SubscriptionProvider
  client={Object {}}
  ws={Object {}}
/>
`;
